src/posix/platform/logging.c

```diff
@@ -30,65 +30,46 @@
 #include "platform-posix.h"
 
 #include <assert.h>
-#include <inttypes.h>
 #include <stdarg.h>
-#include <stdio.h>
 #include <syslog.h>
 
 #include <openthread/platform/logging.h>
 
-#include "code_utils.h"
-
-#define LOGGING_MAX_LOG_STRING_SIZE 512
-
 #if OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED
 void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, ...)
 {
     OT_UNUSED_VARIABLE(aLogRegion);
 
-    char         logString[LOGGING_MAX_LOG_STRING_SIZE];
-    int          charsWritten;
-    va_list      args;
-    int          logLevel;
-    unsigned int offset = 0;
-
-    charsWritten = snprintf(&logString[offset], sizeof(logString), "[%" PRIx64 "] ", gNodeId);
-    otEXPECT_ACTION(charsWritten >= 0, logString[offset] = 0);
-    offset += (unsigned int)charsWritten;
-    otEXPECT_ACTION(offset < sizeof(logString), logString[sizeof(logString) - 1] = 0);
-
-    va_start(args, aFormat);
-    charsWritten = vsnprintf(&logString[offset], sizeof(logString) - offset, aFormat, args);
-    va_end(args);
-
-    otEXPECT_ACTION(charsWritten >= 0, logString[offset] = 0);
+    va_list args;
 
-exit:
     switch (aLogLevel)
     {
     case OT_LOG_LEVEL_NONE:
-        logLevel = LOG_ALERT;
+        aLogLevel = LOG_ALERT;
```

Comment:
Suggestion: how about using an `int logLevel` here instead of re-using `aLogLevel` argument? Mainly since we may in future change `aLogLevel` to be an enumeration for OT log levels and we want to convert it to `systlog` log level.